addons/refuel/functions/fnc_dropNozzle.sqf

```diff
@@ -29,11 +29,16 @@ _nozzle setVelocity [0, 0, 0];
 
 private _groundPosition = getPosASL _nozzle;
 private _posA = (getPosASL _nozzle) vectorAdd [0,0,0.05];
-private _posB = (getPosASL _nozzle) vectorAdd [0,0,-1000];
+private _posB = (getPosASL _nozzle) vectorAdd [0,0,-4];
```

Comment:
Why? Someone could intentionally drop the nozzle from a roof, no?

Reply:
If nozzle drops too far then we end up with weird condition where it's further away than the hose length

Reply:
make the vector max hose length in the Z instead of 4m